integreat_cms/cms/templates/generic_tutorial_dialog.html

```diff
@@ -0,0 +1,26 @@
+{% load i18n %}
+
+<div class="fixed items-center justify-center overflow-scroll inset-0 bg-opacity-75 bg-gray-800 z-50 cursor-pointer {% if hidden %}hidden{% else %}flex{% endif %}" data-tutorial-overlay="{{ tutorial_id }}">
+    <div class="cursor-auto flex flex-col justify-center max-h-fit max-w-5xl w-auto px-10 z-50 m-auto">
+        <div class="bg-white opacity-100 content rounded shadow-md w-full">
+            {% block content %}{% endblock %}
+            {% if tutorial_id %}
+                <form method="post" action="{% url 'dismiss_tutorial' %}" data-tutorial-overlay-form class="p-2">
+                    {% csrf_token %}
+                    <input type="hidden" name="tutorial_id" value="{{ tutorial_id }}" />
+                    <input type="hidden" name="redirect_url" value="{{ request.get_full_path }}" />
+                    
+                    <div class="flex px-2 gap-2 justify-between">
+                        <label for="dismiss-tutorial" class="flex font-normal text-s text-gray-600 items-center">
+                            <input id="dismiss-tutorial" type="checkbox" name="dismiss_tutorial" value="{{ request.user.page_tree_tutorial_seen }}" />
```

Comment:
```suggestion
                            <input id="dismiss-tutorial" type="checkbox" name="dismiss_tutorial" {% if request.user.page_tree_tutorial_seen %}checked{% endif %} />
```
Ok, my previous suggestion didn't work as I expected^^